src/main/java/com/smalaca/messagesender/service/EmailStatsService.java

```diff
@@ -0,0 +1,41 @@
+package com.smalaca.messagesender.service;
+
+import com.smalaca.messagesender.domain.EmailStat;
+import com.smalaca.messagesender.repository.inmemory.MessageStatsRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class EmailStatsService {
+
+    private MessageStatsRepository messageStatsRepository;
```

Comment:
Serwis nie może zależeć od JPA-based interfejsu. To musi być coś nad czym Wy macie kontrolę.